File: ProjectTracker20vb/PTWin/LoginForm.vb
Public Class LoginForm

  Private Sub OK_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles OK.Click

    Using busy As New StatusBusy("Verifying credentials...")
      ProjectTracker.Library.Security.PTPrincipal.Login( _
          Me.UsernameTextBox.Text, Me.PasswordTextBox.Text)
    End Using
    Me.Close()

  End Sub

  Private Sub Cancel_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Cancel.Click

    Me.Close()

  End Sub

  Private Sub LoginForm_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load

    Me.UsernameTextBox.Focus()

  End Sub

End Class

File: Samples/ProjectTracker/ProjectTracker.Ui.WisejWeb/LoginForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using Wisej.Web;

namespace PTWisej
{
  public partial class LoginForm : Form
  {
    public LoginForm()
    {
      InitializeComponent();
    }

    private void OK_Click(object sender, EventArgs e)
    {
      using (StatusBusy busy = 
        new StatusBusy("Verifying credentials..."))
      {
        ProjectTracker.Library.Security.PTPrincipal.Login(
          this.UsernameTextBox.Text, this.PasswordTextBox.Text);
      }
      this.Close();
    }

    private void Cancel_Click(object sender, EventArgs e)
    {
      this.Close();
    }

    private void LoginForm_Load(object sender, EventArgs e)
    {
      this.UsernameTextBox.Focus();
    }
  }
}